src/SprykerEco/Service/AkeneoPim/AkeneoPimServiceInterface.php

```diff
@@ -537,4 +539,401 @@ public function getProductMediaFilesListPerPage($limit = 10, $withCount = false,
      * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
      */
     public function getProductModelsListPerPage($limit = 10, $withCount = false, array $queryParameters = []): AkeneoResourcePageInterface;
+
+    /**
+     * Specification:
+     *  - Gets a cursor to iterate over a list of assets.
+     *
+     * @api
+     *
+     * @param int $pageSize
+     * @param array $queryParameters
+     *
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourceCursorInterface
+     */
+    public function getAllAssets(int $pageSize = 10, array $queryParameters = []): AkeneoResourceCursorInterface;
+
+    /**
+     * Specification:
+     *  - Gets an asset by its code
+     *
+     * @api
+     *
+     * @param string $code
+     *
+     * @return array
+     */
+    public function getAsset(string $code): array;
+
+    /**
+     * Specification:
+     *  - Gets a list of assets by returning the first page.
+     *
+     * @api
+     *
+     * @param int $limit
+     * @param bool $withCount
+     * @param array $queryParameters
+     *
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
+     */
+    public function getAssetsListPerPage(int $limit = 10, bool $withCount = false, array $queryParameters = []): AkeneoResourcePageInterface;
+
+    /**
+     * Specification:
+     *  - Gets a cursor to iterate over a list of asset tags.
+     *
+     * @api
+     *
+     * @param int $pageSize
+     * @param array $queryParameters
+     *
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourceCursorInterface
+     */
+    public function getAllAssetTags(int $pageSize = 10, array $queryParameters = []): AkeneoResourceCursorInterface;
+
+    /**
+     * Specification:
+     *  - Gets an asset tag by its code
+     *
+     * @api
+     *
+     * @param string $code
+     *
+     * @return array
+     */
+    public function getAssetTag(string $code): array;
+
+    /**
+     * Specification:
+     *  - Gets a list of asset tags by returning the first page.
```

Comment:
Don't really get this specification

Reply:
I copied the existing one from similar methods. I'm not sure this one should have completely different description